Assets/DevDunkStudio.meta

```diff
@@ -1,8 +0,0 @@
-fileFormatVersion: 2
-guid: e997a26d94ce4b6419d45920082a800c
-folderAsset: yes
-DefaultImporter:
-  externalObjects: {}
-  userData: 
-  assetBundleName: 
-  assetBundleVariant: 
```

Comment:
I don't quite understand what shaders and materials are for in this submodule

Reply:
That's the shadow receiver - which is a for-pay package.